tasks/__init__.py

```diff
@@ -66,7 +66,12 @@ def server(host=None, port=5000, debug=True, live=False, gitlogs=False):
         server.watch(os.path.join(HERE, 'website', 'static', 'public'))
         server.serve(port=port)
     else:
-        app.run(host=host, port=port, debug=debug, threaded=debug, extra_files=[settings.ASSET_HASH_PATH])
+        if settings.SECURE_MODE:
+            context = (settings.OSF_SERVER_CERT, settings.OSF_SERVER_KEY)
+        else:
+            context = None
+        app.run(host=host, port=port, debug=debug, threaded=debug, extra_files=[settings.ASSET_HASH_PATH], ssl_context=context)
```

Comment:
As we discussed, it is OK for us to use Flask `ad_hoc` to generate certs on the fly. However, this will cause the browser raise trust dialog overtime server restarts. In additions, django-ssl server have a hardcoded cert which is very dangerous to trust.